pkg/scheduler/core/generic_scheduler.go

```diff
@@ -286,21 +282,22 @@ func (g *genericScheduler) Preempt(ctx context.Context, prof *profile.Profile, s
 		klog.Infof("%v potential nodes for preemption, first %v are: %v", len(potentialNodes), len(sample), sample)
 	}
 	var pdbs []*policy.PodDisruptionBudget
-	if g.pdbLister != nil {
```

Comment:
Can pdbLister now be removed from genericScheduler?

Reply:
Yes, let me remove it.